paddle/fluid/pybind/cuda_streams_py.cc

```diff
@@ -200,14 +212,16 @@ void BindCudaStream(py::module *m_ptr) {
                    "Priority should be 1(high) or 2(normal) "));
              }
              auto prio = paddle::platform::stream::Priority(priority);
+             auto stream_flag = paddle::platform::stream::StreamFlag(1);
 
```

Comment:
What's the hard code `1` means?

Reply:
1 means non-blocking stream. We init CUDA Stream with default non-blocking property following pytorch implementation.

Reply:
How about using `paddle::platform::stream::StreamFlag::kStreamNonBlocking` instead of `1`?

Reply:
done